### PR TITLE
Updated instructions and launch files

### PR DESCRIPTION
**Pull request by batikim09**
Updated for a new mic and demos.

### CODE REVIEW

File: squirrel_ser/src/cobotiny_arousal.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+import argparse
+import sys
+import rospy
+import numpy as np
+import copy
+from std_msgs.msg import Int32,Float32,String
+from visualization_msgs.msg import Marker, MarkerArray
```
Comment (NadiaHG) [inline squirrel_ser/src/cobotiny_arousal.py:8]: please add missed dependency to https://github.com/squirrel-project/squirrel_hri/blob/indigo_dev/squirrel_ser/package.xml

File: squirrel_ser/src/cobotiny_arousal.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+from tf.transformations import quaternion_from_euler
+from geometry_msgs.msg import Quaternion
```
Comment (NadiaHG) [inline squirrel_ser/src/cobotiny_arousal.py:10]: please add missed dependency to https://github.com/squirrel-project/squirrel_hri/blob/indigo_dev/squirrel_ser/package.xml

File: squirrel_ser/src/cobotiny_arousal.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+import argparse
+import sys
+import rospy
+import numpy as np
+import copy
+from std_msgs.msg import Int32,Float32,String
+from visualization_msgs.msg import Marker, MarkerArray
+from tf.transformations import quaternion_from_euler
```
Comment (NadiaHG) [inline squirrel_ser/src/cobotiny_arousal.py:9]: please add missed dependency to https://github.com/squirrel-project/squirrel_hri/blob/indigo_dev/squirrel_ser/package.xml

File: squirrel_ser/src/cobotiny_arousal.py (hunk 2 above)
Comment (batikim09) [reply squirrel_ser/src/cobotiny_arousal.py:10]: I added dependencies in the following commit. Let me know anything missing. Thanks!

File: squirrel_ser/package.xml [hunk 4]
```diff
@@ -17,9 +17,16 @@
   <build_depend>rospy</build_depend>
   <build_depend>std_msgs</build_depend>
   <build_depend>squirrel_vad_msgs</build_depend>
+  <build_depend>visualization_msgs</build_depend>
+  <build_depend>geometry_msgs</build_depend>
+  <build_depend>tf.transformations</build_depend>
```
Comment (NadiaHG) [inline squirrel_ser/package.xml]: @batikim09  thanks! just a small comment , the name of the ROS package is `tf` and not `tf.transformations`

**Review by ipa-nhg — APPROVED** [review]

**Pull request merged**